### DISCUSSION TITLE
Server Sent Events

### DISCUSSION THREAD

**Discussion by dekakast** [AzuraCast/AzuraCast#6947] [Q&A]
I am not connecting with the code below and wondering if my azuracast url path is wrong, or if I need to enable something in my server system settings. Appreciate any suggestions., especially @Moonbase59 if you see this :)  Obviously, I edit my "azuracast_url" and "azuracast_station_id"

`
// Updated SSE connection setup with URL parameters for subscriptions
        const sseBaseUri = "https://azuracast_url/api/live/nowplaying/sse"; // Update the base URL
        const sseUriParams = new URLSearchParams({
          "cf_connect": JSON.stringify({
            "subs": {
              "station:azuratest_station_id": {}, // Make sure to update this to your actual station ID
              "global:time": {}
            }
          })
        });
        const sseUri = `${sseBaseUri}?${sseUriParams.toString()}`;
        const sse = new EventSource(sseUri);
`

**Comment by Moonbase59** [comment]
Sure I see this ;-) See https://gist.github.com/Moonbase59/d42f411e10aff6dc58694699010307aa?permalink_comment_id=4909032#gistcomment-4909032.

Where exactly are you stuck?
